source/common/http/http1/codec_impl.h

```diff
@@ -363,6 +365,9 @@ class ClientConnectionImpl : public ClientConnection, public ConnectionImpl {
   std::list<PendingResponse> pending_responses_;
   // Set true between receiving 100-Continue headers and receiving the spurious onMessageComplete.
   bool ignore_message_complete_for_100_continue_{};
+
+  // The default limit of 80 KiB is the vanilla http_parser behaviour.
```

Comment:
The default limit is 80Kb (Kilobytes) not KiB (Kibibytes), no?

Reply:
Oh jeez, trusted the http_parser issue which was incorrect  - looks like they were saying KB when they meant KiB.  You can ignore this one.